composer.json

```diff
@@ -40,16 +40,16 @@
         "symfony/twig-bundle": "^4.4 || ^5.1",
         "symfony/validator": "^4.4 || ^5.1",
         "symfony/yaml": "^4.4 || ^5.1",
-        "webmozart/assert": "^1.8",
-        "winzou/state-machine-bundle": "^0.3.2 || ^0.4.3 || ^0.5"
+        "webmozart/assert": "^1.8"
     },
     "replace": {
         "sylius/resource": "self.version"
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<3.0 >=4.0",
         "jms/serializer-bundle": "<3.5 >=4.0",
-        "willdurand/hateoas-bundle": "<2.0 >=3.0"
+        "willdurand/hateoas-bundle": "<2.0 >=3.0",
+        "winzou/state-machine-bundle": "<0.3.2 || >=0.4.0 <0.4.3"
```

Comment:
@pamil I'm not sure at all for "<0.3.2 || >=0.4.0 <0.4.3", Can you confirm it?

Reply:
Should be fine I guess 👍